geomesa-accumulo/geomesa-accumulo-tools/bin/setup-namespace.sh

```diff
@@ -23,21 +24,25 @@ function help() {
   echo ""
   echo "Optional:"
   echo "  -p    Accumulo password for the provided username"
+  echo "  -t    Login using an existing Kerberos token, else will prompt for password"
   echo "  -g    Path of the GeoMesa distributed runtime JAR, with or without raster support."
   echo "  -d    Directory to create namespace in, default: /accumulo/classpath"
   echo "  -h    HDFS URI e.g. hdfs://localhost:54310"
   echo ""
   exit 0
 }
 
-while getopts ":u:p:n:g:h:d:help" opt; do
+while getopts ":u:p:tn:g:h:d:help" opt; do
```

Comment:
From http://stackoverflow.com/questions/18414054/bash-getopts-reading-optarg-for-optional-flags/18414091#18414091 I *thought* flag type parameters (which are either present or absent i.e. don't take a parameter) didn't use a colon. However, that link isn't exactly definitive